Sources/UIKitBackend/UIViewRepresentable.swift

```diff
@@ -20,12 +21,14 @@ where Content == Never {
     ///   - context: The context, including the coordinator and potentially new environment
     ///   values.
     /// - Note: This may be called even when `context` has not changed.
+    @MainActor
     func updateUIView(_ uiView: UIViewType, context: UIViewRepresentableContext<Coordinator>)
 
     /// Make the coordinator for this view.
     ///
     /// The coordinator is used when the view needs to communicate changes to the rest of
     /// the view hierarchy (i.e. through bindings), and is often the view's delegate.
+    @MainActor
     func makeCoordinator() -> Coordinator
```

Comment:
`determineViewSize` should probably be `@MainActor` too (it's not close enough to your changed lines for me to comment on it directly)